var/spack/repos/builtin/packages/cube/package.py

```diff
@@ -46,16 +51,19 @@ class Cube(Package):
     variant('gui', default=False, description='Build CUBE GUI')
 
     depends_on('zlib')
-    depends_on('qt@4.6:', when='+gui')
 
-    def install(self, spec, prefix):
-        configure_args = ["--prefix=%s" % prefix,
-                          "--without-paraver"]
```

Comment:
The way that some Autotools and CMake package work is that by default, they will build paraver support if they can find paraver installed, otherwise they don't. This is kind of non-deterministic and results in unreproducible builds. In this case, I think it's better to explicitly disable it until someone adds a paraver package as a dependency and makes it an optional variant.